rules/allied-infantry.yaml

```diff
@@ -76,13 +75,11 @@ dog:
 		VoiceSet: DogVoice
 	ProducibleWithLevel:
 		Prerequisites: barracks.infiltrated
-	-TakeCover:
```

Comment:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/Infantry/TakeCover.cs is still available and shouldn't be removed for the dog.